salt/modules/dockercompose.py

```diff
@@ -769,3 +889,137 @@ def up(path, service_names=None):
+    services = compose_result['compose_content']['services']
+    if service_name in services:
+        msg = 'Service {0} already exists'.format(service_name)
+        return __standardize_result(False, msg, None, None)
+    services[service_name] = loaded_definition
+    return __dump_compose_file(path, compose_result,
+                               'Service definition for {0} is set'.format(service_name),
+                               already_existed=True)
+
+
+def service_remove(path, service_name):
+    '''
+    Remove the definition of a docker-compose service
+    This does not rm the container
+    This wil re-write your yaml file. Comments will be lost. Indentation is set to 2 spaces
+
+    path
+        Path where the docker-compose file is stored on the server
+    service_name
+        Name of the service to remove
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt myminion dockercompose.service_remove /path/where/docker-compose/stored service_name
+    '''
+    compose_result, err = __load_docker_compose(path)
+    if err:
+        return err
+    services = compose_result['compose_content']['services']
+    if service_name not in services:
+        return __standardize_result(False,
+                                    'Service {0} did not exists'.format(service_name),
+                                    None, None)
+    del services[service_name]
+    return __dump_compose_file(path, compose_result,
+                               'Service {0} is removed from {1}'.format(service_name, path),
+                               already_existed = True)
+
+
+def service_set_tag(path, service_name, tag):
+    '''
+    Change the tag of a docker-compose service
+    This does not pull or up the service
+    This wil re-write your yaml file. Comments will be lost. Indentation is set to 2 spaces
+
+    path
+        Path where the docker-compose file is stored on the server
+    service_name
+        Name of the service to remove
+    tag
+        Name of the tag (often used as version) that the service image should have
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt myminion dockercompose.service_create /path/where/docker-compose/stored service_name tag
+    '''
+    compose_result, err = __load_docker_compose(path)
+    if err:
+        return err
+    services = compose_result['compose_content']['services']
+    if service_name not in services:
+        return __standardize_result(False,
+                                    'Service {0} did not exists'.format(service_name),
+                                    None, None)
+    if 'image' not in services[service_name]:
+        return __standardize_result(False,
+                                    'Service {0} did not contain the variable "image"'.format(service_name),
+                                    None, None)
+    image = services[service_name]['image'].split(':')[0]
+    services[service_name]['image'] = str('{0}:{1}'.format(image, tag))
```

Comment:
If you use `safe_dump` above, you can replace this with `'{0}:{1}'.format(image, tag)`.